README.md

```diff
@@ -13,7 +13,7 @@ The following features will be included in future releases.
 - Intel® In-Memory Analytics Accelerator (Intel® IAA) 
```

Comment:
Can we change the commit log title to:
Readme: update root readme

It would be great if we could change the PR tile at the same time.

Reply:
Sure, done!

Reply:
Looks good to me.